content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: MedicationAdministration | DSTU 2 API
+---
+
+# MedicationAdministration
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Administration resource is a record of the event of a patient consuming or otherwise being administered a medication.  Medications Administrations are assumed to be Given unless documented otherwise.
```

Comment:
I think we may have a different/more expansive description we can use that details a bit more about what an admin may include.

Reply:
Added the approved description: https://github.com/cerner/fhir.cerner.com/pull/169/commits/037712040917c1d556c6c8c76363db6eb41412c2#diff-39322533700e1bb799f2f6d703a9ae74L12